roles/etcd_ca/defaults/main.yml

```diff
@@ -0,0 +1,2 @@
+---
+openshift_etcd_ca_host: "{{ groups.oo_first_etcd.0 }}"
```

Comment:
I'm not sure we want to rely on specific group names at the role level, the roles should really be self-sufficient and testable without needing to expose playbook specific details.

Instead, I would say that we just require that openshfit_etcd_ca_host be set, and pass that in as a variable to the role.